pipelines/tekton/gitops-update-pipeline/example-pipelineruns/gitops-update-pipelinerun-json.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  generateName: gitops-update-pipeline-test-json-
+  labels:
+    tekton.dev/pipeline: gitops-update-pipeline
+spec:
+  params:
+  - name: model-name
+    value: tensorflow-housing
+  - name: gitServer
+    value: https://github.com
+  - name: gitApiServer
+    value: api.github.com
+  - name: gitOrgName
+    value: opendatahub-io
+  - name: gitRepoName
+    value: ai-edge
+  - name: gitRepoBranchBase
+    value: main
+  - name: gitTokenSecretName
+    value: edge-user-1
+  - name: yq-script
+    value: |
+      yq eval -i -o=json -I=4 "(.spec.template.spec.containers[] | select(.name == \"model\").image) = \"${IMAGE_NAME}@${IMAGE_DIGEST}\"" \
+        pipelines/tekton/gitops-update-pipeline/test/json/my-deployment.json
```

Comment:
This example won't work as-is until this file exists on the target branch (gitRepoBranchBase).

To try it out before merging, one coud set gitOrgName to grdryn and gitRepoBranchBase to RHOAIENG-3128_json-test. An example of a PR created by this can be seen https://github.com/grdryn/ai-edge/pull/15